mock-thunderhead/src/main/resources/application.yml

```diff
@@ -34,4 +34,4 @@ auth:
     database.wire.encryption.enable: true
     datahub.runtime.upgrade.enable: true
     embedded.database.on.attached.disk.enable: true
-    datalake.loadbalancer.enable: false
+    datalake.loadbalancer.enable: true
```

Comment:
I've been asked by the CB team to not enable this by default, since it only really works if you're using mow-dev UMS and most people won't be.